packages/app/src/components/Settings/Settings.js

```diff
@@ -194,59 +214,114 @@ const Settings = (props) => {
           <div className="settings__input-block">
             <p className="settings__input-label">Start Date:</p>
             <span className="settings__input">
-              <input
-                value={newGrantCycle.openedOn}
-                name="openedOn"
-                onChange={handleChange}
-                type="date"
-              />
+              <div
+                style={
+                  showEditModal
+                    ? {
+                        display: 'none',
+                      }
+                    : {
+                        position: 'relative',
+                      }
+                }
```

Comment:
I'd prefer not to use inline styles. Instead, I think `showEditModal` we should toggle on/off a `.hidden` class that has `display: 'none'` in the CSS.